sklearn/plot/_gridsearch_results.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
+from sklearn.plot import plot_heatmap
+
+
+def plot_gridsearch_results(cv_results, metric='mean_test_score',
```

Comment:
It's the most common case, I'd say. Maybe check if you could add it to an example doing ``validation_curve``?